<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reser</class>
 <widget class="QDialog" name="reser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="hotel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>591</width>
     <height>311</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="input_hotel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>500</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_attraction">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>450</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="guide">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Guide</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>251</height>
    </rect>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Spot">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attraction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slt_btn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>전 환</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_at">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>예약관리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fixspot">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>450</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>관광지수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_btn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_guide">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>550</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>441</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delspot">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>450</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>예약취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delhotel">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>500</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>예약취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fixhotel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>500</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>호텔수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fixguide">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>550</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delguide">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>550</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>예약취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>입력 후 수정 하세요</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fixdate">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>590</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>날짜수정</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_date">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delall">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>630</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>전체 취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
